packages/devtools_app/lib/src/screens/inspector/layout_explorer/flex/flex.dart

```diff
@@ -540,34 +547,36 @@ class FlexChildVisualizer extends StatelessWidget {
 
   final RenderProperties renderProperties;
 
-  FlexLayoutProperties get root => state.properties;
+  FlexLayoutProperties? get root => state.properties;
```

Comment:
make these both non-nullable even if it means adding a bang. 
Consider adding a TODO to refactor code to eliminate the bang. The core logic of this visualizer doesn't make sense if these are null.

Reply:
done